domain/schema/model/sql/0004-read-only-model.sql

```diff
@@ -22,3 +22,9 @@ CREATE TABLE model (
 -- A unique constraint over a constant index ensures only 1 entry matching the
 -- condition can exist.
 CREATE UNIQUE INDEX idx_singleton_model ON model ((1));
+
+CREATE VIEW v_model_metrics AS
+SELECT
+    (SELECT COUNT(DISTINCT uuid) FROM application) AS application_count,
+    (SELECT COUNT(DISTINCT uuid) FROM machine) AS machine_count,
+    (SELECT COUNT(DISTINCT uuid) FROM unit) AS unit_count;
```

Comment:
uuids here are primary keys, so we could just have the simpler `SELECT COUNT(*) FROM ...` instead right?